### PR TITLE
User docs changes in WGE install guide, cluster mgmt pages

### PR DESCRIPTION
**Pull request by LappleApple**
**What changed?**

- Removes almost all pages in "Access Configuration," the contents of which are now published (and updated) in the main WGE install guide. 
- Addresses link changes in current and v0.30.0 versions. 
- Addresses feedback from CX and Engineering related to missing or inaccurate information in some Cluster Mgmt pages.

**Why was this change made?**

To fix errors, gaps, and other docs issues

**How was this change implemented?**

Substantial changes triggered by feedback from Sales, Eng, and other internal parties about issues using the user docs.

**How did you validate the change?**

Ran npm run build several times to fix broken links

**Release notes**

Fixes errors and gaps in release guide Cluster Management and WGE Install guide

**Documentation Changes**

The PR is a doc change

### CODE REVIEW

File: website/docs/cluster-management/cluster-management-intro.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ In line with the mantra “cattle, not pets,” Weave GitOps Enterprise (WGE) si
 WGE fully supports a range of options, including:
 - [Crossplane integration](https://www.weave.works/blog/gitops-goes-beyond-kubernetes-with-weave-gitops-upbound-s-universal-crossplane)
 - Terraform integration, with a [Terraform Controller](https://docs.gitops.weave.works/docs/terraform/overview/) that follows the patterns established by Flux
-- Cluster API 
+- [Cluster API](https://cluster-api.sigs.k8s.io/)
```
Comment (Himangini) [inline website/docs/cluster-management/cluster-management-intro.mdx:15]: should these changes go in `/version-0.30.0` as well?

File: website/docs/cluster-management/managing-clusters-without-capi.mdx [hunk 2]
```diff
@@ -154,7 +153,13 @@ Here's how to create a kubeconfig secret.
 
 	</details>
 
-5. Obtain the cluster certificate (CA). How you do this depends on your cluster. If you're using GKE, for example, you can view the CA on the GCP Console: Cluster->Details->Endpoint->”Show cluster certificate”. You'll need to copy the contents of the certificate into the `ca.crt` file used below.
+5. Obtain the cluster certificate (CA). How you do this depends on your cluster.
```
Comment (Himangini) [inline website/docs/cluster-management/managing-clusters-without-capi.mdx:156]: should these changes go in `/version-0.30.0` as well?

File: website/docs/enterprise/getting-started/releases-enterprise.mdx [hunk 3]
```diff
@@ -50,7 +50,7 @@ We introduced a breaking change in this release by upgrading to Flux v2 APIs, no
 
 ### ✍️ Action required
 
-Follow [Flux](https://github.com/fluxcd/flux2/releases/tag/v2.0.0) or [Weave Gitops](https://docs.gitops.weave.works/docs/guides/fluxga-upgrade/) to upgrade to Flux v2 GA before upgrading Weave Gitops Enterprise.
+Follow [Flux](https://github.com/fluxcd/flux2/releases/tag/v2.0.0) or [Weave GitOps](https://docs.gitops.weave.works/docs/guides/fluxga-upgrade/) to upgrade to Flux v2 GA before upgrading Weave GitOps Enterprise.
```
Comment (Himangini) [inline website/docs/enterprise/getting-started/releases-enterprise.mdx:53]: should these changes on this page go in `/version-0.30.0` as well?

File: website/docs/cluster-management/cluster-management-intro.mdx (hunk 1 above)
Comment (LappleApple) [reply website/docs/cluster-management/cluster-management-intro.mdx:15]: Would be a good idea. I'll add.

File: website/docs/cluster-management/managing-clusters-without-capi.mdx [hunk 4]
```diff
@@ -12,9 +12,8 @@ import BrowserOnly from "@docusaurus/BrowserOnly";
 
 # Managing Clusters Without Cluster API <TierLabel tiers="Enterprise" />
 
-You do **not** need Cluster API to add your Kubernetes cluster to Weave Gitops Enterprise. The only thing you need is a secret containing a valid `kubeconfig`.
+You do **not** need Cluster API to add your Kubernetes cluster to Weave GitOps Enterprise. The only thing you need is a secret containing a valid `kubeconfig`.
 
-import TOCInline from "@theme/TOCInline";
```
Comment (Himangini) [inline website/docs/cluster-management/managing-clusters-without-capi.mdx:17]: this shouldn't be removed I think or line 18 should also be removed with this
Screenshot 2023-08-18 at 13 12 51

File: website/docs/enterprise/getting-started/install-enterprise.mdx [hunk 5]
```diff
@@ -628,7 +628,7 @@ various namespaces is also cached to speed up future operations.
 
 The `cluster-user-auth` and `oidc-auth` secrets provide information for authenticating
 to the application. The former holds the username and bcrypt-hashed password
-for the [emergency user](../../configuration/emergency-user.mdx), and the latter holds OIDC configuration.
+for the [emergency user](#configuring-the-emergency-user), and the latter holds OIDC configuration.
```
Comment (Himangini) [inline website/docs/enterprise/getting-started/install-enterprise.mdx:631]: same here

File: website/docs/cluster-management/managing-clusters-without-capi.mdx (hunk 2 above)
Comment (LappleApple) [reply website/docs/cluster-management/managing-clusters-without-capi.mdx:156]: Done

File: website/docs/enterprise/getting-started/install-enterprise.mdx [hunk 6]
```diff
@@ -604,7 +604,7 @@ The primary way Weave GitOps queries the Kube API is via `impersonation`. The pe
 can impersonate will determine the scope of actions that WGE can take within your cluster. 
 
 The application, not the cluster, authenticates the user, either via the [emergency
-cluster user](../../configuration/emergency-user.mdx) credentials or OIDC. Then it makes Kube API calls on the user's
+cluster user](#configuring-the-emergency-user) credentials or OIDC. Then it makes Kube API calls on the user's
```
[the remark this answers is not in the document]
Comment (LappleApple) [reply website/docs/enterprise/getting-started/install-enterprise.mdx:607]: @Himangini We did that [here](https://docs.gitops.weave.works/docs/enterprise/getting-started/install-enterprise/#configuring-the-emergency-user) in an earlier PR. It's a sub-section of the same page, and I seem to be formatting the link to it improperly.

File: website/docs/cluster-management/managing-clusters-without-capi.mdx (hunk 4 above)
Comment (Himangini) [reply website/docs/cluster-management/managing-clusters-without-capi.mdx:17]: yeah if line 17 changes were not intentional then we should add it back.

File: website/docs/enterprise/getting-started/install-enterprise.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (Himangini) [reply website/docs/enterprise/getting-started/install-enterprise.mdx:607]: I can't see the text `configuring emergency user` at all on our live docs. All I see is the screenshot below when I click the link from your comment https://docs.gitops.weave.works/docs/enterprise/getting-started/install-enterprise/#configuring-the-emergency-user

Screenshot 2023-08-18 at 13 46 23

File: website/docs/enterprise/getting-started/install-enterprise.mdx (hunk 5 above)
Comment (Himangini) [reply website/docs/enterprise/getting-started/install-enterprise.mdx:631]: can you add a screenshot of the section from the live docs? i can't see it at all 🙈

File: website/docs/enterprise/getting-started/install-enterprise.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (LappleApple) [reply website/docs/enterprise/getting-started/install-enterprise.mdx:607]: here's a fresh screenshot from my side 
configuring emergency user

File: website/docs/enterprise/getting-started/install-enterprise.mdx (hunk 5 above)
Comment (LappleApple) [reply website/docs/enterprise/getting-started/install-enterprise.mdx:631]: I did in the last comment -- wild!

File: website/docs/cluster-management/managing-clusters-without-capi.mdx (hunk 4 above)
Comment (LappleApple) [reply website/docs/cluster-management/managing-clusters-without-capi.mdx:17]: done

File: website/docs/enterprise/getting-started/install-enterprise.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (Himangini) [reply website/docs/enterprise/getting-started/install-enterprise.mdx:607]: Ok my bad, this is under the 3rd tab labeled `Login via a cluster user account`. I kept looking at the first tab lols 🙈

**Comment by LappleApple** [comment]
@Himangini Thanks for taking a look -- have you completed your review?

**Review by Himangini — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**